jax/_src/numpy/ufuncs.py

```diff
@@ -1984,9 +1984,41 @@ def frexp(x: ArrayLike, /) -> tuple[Array, Array]:
   return _where(cond, x, x1), lax.convert_element_type(x2, np.int32)
 
 
-@implements(np.remainder, module='numpy')
 @jit
 def remainder(x1: ArrayLike, x2: ArrayLike, /) -> Array:
+  """Returns element-wise remainder of the division.
+
+  JAX implementation of :obj:`numpy.remainder`.
+
+  The remainder is computed by ``x1 - x2 * jnp.floor(x1 / x2)``.
```

Comment:
I don't think this is correct in terms of the actual implementation.

Reply:
Thanks! Removed the statement and made a note about it that they equivalent.